exporters/otlp/README.md

```diff
@@ -0,0 +1,25 @@
+# OTLP Exporter
+
+The [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/README.md) (OTLP) is a vendor-agnostic protocol designed as part of the OpenTelemetry project. The OTLP exporter can be used to export to any backend that supports OTLP.
+
+Currently, the only backend that supports OTLP is the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector). The Collector can be configured to export to many other backends, such as Zipkin and Jaegar.
+
+For a full list of backends supported by the Collector, see the [main Collector repo](https://github.com/open-telemetry/opentelemetry-collector/tree/master/exporter) and [Collector contributions](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/exporter).
+
+## Configuration
+The OTLP exporter offers some configuration options. To configure the exporter, create an `OtlpExporterOptions` struct (defined in [exporter.h](include/exporter.h)), set the options inside, and pass the struct to the `OtlpExporter` constructor, like so:
+
+```
+OtlpExporterOptions options;
+options.endpoint = "localhost:12345";
+auto exporter = std::unique_ptr<sdktrace::SpanExporter>(new otlp::OtlpExporter(options));
+```
+
+### Configuration options
+
+| Option       | Default           |
+| ------------ |------------------ |
+| `endpoint`   | `localhost:55680` |
+
+## Example
+For a complete example demonstrating how to use the OTLP exporter, see [examples/otlp](../../examples/otlp).
```

Comment:
nit: probably add an empty line after level 2 heading - you can also try markdownlint.